### PR TITLE
[Naitei][Feature][User] View bill detail

### PR DESCRIPTION
**Pull request by TranMinhHuy2003**
## Related Tickets
[[User] Xem chi tiết hóa đơn](https://edu-redmine.sun-asterisk.vn/issues/78826)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
passrubocop_view_bill_detail
view_bill_detail_en
view_bill_detail_vi

### CODE REVIEW

**Comment by TranMinhHuy2003** [comment]
ready

File: app/views/bills/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            <p><%= t "cart.quantity" %></p>
+            <p><%= t "cart.total" %></p>
+            <p><%= t "order_history.review" %></p>
+          </div>
+          <div class="products__list">
+            <%= render @bill_details %>
+          </div>
+        </div>
+      </div>
+      <% if @bill.status == "wait_for_pay" %>
```
Comment (ghost) [inline app/views/bills/show.html.erb]:
```suggestion
      <% if @bill.wait_for_pay? %>
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by TranMinhHuy2003** [comment]
ready

**Pull request merged**